fe/lib/utils/formatPoint.ts

```diff
@@ -0,0 +1,7 @@
+export const formatPoint = (point: [number, number]): string => {
+  const roundedLat = Math.abs(Math.round(point[0] * 100) / 100);
+  const roundedLon = Math.abs(Math.round(point[1] * 100) / 100);
+  const lat = roundedLat < 0 ? `${roundedLat}° S` : `${roundedLat}° N`;
+  const lon = roundedLon < 0 ? `${roundedLon}° W` : `${roundedLon}° E`;
```

Comment:
Is this logic correct? It seems like `roundedX < 0` will always return false given the `Math.abs`.

Would we rather want something like this

```suggestion
export const formatPoint = (point: [number, number]): string => {
  const roundedLat = Math.abs(point[0]).toFixed(2);
  const roundedLon = Math.abs(point[1]).toFixed(2);
  const lat = point[0] < 0 ? `${roundedLat}° S` : `${roundedLat}° N`;
  const lon = point[1] < 0 ? `${roundedLon}° W` : `${roundedLon}° E`;
```

Could be a good util to unit test too 🙂

Reply:
Oops yep you're definitely right with that, will use this suggested solution, thanks! 😄